components/match2/wikis/mobilelegends/matchgroup_input_custom.lua

```diff
@@ -0,0 +1,651 @@
+	match.liquipediatier = Logic.emptyOr(match.liquipediatier, Variables.varDefault('tournament_liquipediatier'))
+	match.liquipediatiertype = Logic.emptyOr(
+		match.liquipediatiertype,
+		Variables.varDefault('tournament_liquipediatiertype')
+	)
+	match.publishertier = Logic.emptyOr(match.publishertier, Variables.varDefault('tournament_publishertier'))
+	return match
+end
+
+function matchFunctions.getVodStuff(match)
+	match.stream = match.stream or {}
+	match.stream = {
+		stream = Logic.emptyOr(match.stream.stream, Variables.varDefault('stream')),
+		twitch = Logic.emptyOr(match.stream.twitch or match.twitch, Variables.varDefault('twitch')),
+		twitch2 = Logic.emptyOr(match.stream.twitch2 or match.twitch2, Variables.varDefault('twitch2')),
+		nimo = Logic.emptyOr(match.stream.nimo or match.nimo, Variables.varDefault('nimo')),
+		trovo = Logic.emptyOr(match.stream.trovo or match.trovo, Variables.varDefault('trovo')),
+		huya = Logic.emptyOr(match.stream.huya or match.huya, Variables.varDefault('huya')),
+		text = Logic.emptyOr(match.stream.text or match.text, Variables.varDefault('text')),
+		separator = Logic.emptyOr(match.stream.separator or match.twitch2, Variables.varDefault('separator')),
+		afreeca = Logic.emptyOr(match.stream.afreeca or match.afreeca, Variables.varDefault('afreeca')),
+		afreecatv = Logic.emptyOr(match.stream.afreecatv or match.afreecatv, Variables.varDefault('afreecatv')),
+		dailymotion = Logic.emptyOr(match.stream.dailymotion or match.dailymotion, Variables.varDefault('dailymotion')),
+		douyu = Logic.emptyOr(match.stream.douyu or match.douyu, Variables.varDefault('douyu')),
+		smashcast = Logic.emptyOr(match.stream.smashcast or match.smashcast, Variables.varDefault('smashcast')),
+		youtube = Logic.emptyOr(match.stream.youtube or match.youtube, Variables.varDefault('youtube')),
+		facebook = Logic.emptyOr(match.stream.facebook or match.facebook, Variables.varDefault('facebook')),
+	}
+	match.vod = Logic.emptyOr(match.vod, Variables.varDefault('vod'))
+
+	match.lrthread = Logic.emptyOr(match.lrthread, Variables.varDefault('lrthread'))
+
+	match.links = {}
+	local links = match.links
+	if match.reddit then links.reddit = match.reddit end
+	if match.matchhistory then links.matchhistory = match.matchhistory end
+	local historyIndex = 1
+	while String.isNotEmpty(match['matchhistory' .. historyIndex]) do
+		links['matchhistory' .. historyIndex] = match['matchhistory' .. historyIndex]
+		historyIndex = historyIndex + 1
+	end
+	if match.interview then links.interview = match.interview end
+	if match.review then links.review = match.review end
+	if match.recap then links.recap = match.recap end
+
+	-- apply vodgames
+	for index = 1, _MAX_NUM_VODGAMES do
+		local vodgame = match['vodgame' .. index]
+		if not Logic.isEmpty(vodgame) then
+			local map = match['map' .. index] or {}
+			map.vod = map.vod or vodgame
+			match['map' .. index] = map
+		end
+	end
```

Comment:
Is this needed?

Reply:
Yeah I believe this allows adding |vod= within the {{Map}} template, and the vod number icon will matched based on where it appears within **mapX** (only display wise, code writing wise I have no idea) 

I'll be matching this in the 6th update

Reply:
If I'm reading the code correctly, it will allow this format input 
```
{{Match
   ...
   |vodgame1=ABC
   |map1={{Map .... }}
}}```
to be an alias for this
```
{{Match
   ...
   |map1={{Map|vod=ABC.... }}
}}```

Reply:
understood, yeah only the latter that I'm using. Will remove the mentioned codes in next update.